pkg/cmd/cli/cmd/newapp.go

```diff
@@ -142,6 +142,7 @@ func NewCmdNewApplication(fullName string, f *clientcmd.Factory, out io.Writer)
 		},
 	}
 
+	cmd.Flags().BoolVar(&config.AsTestDeployment, "as-test", config.AsTestDeployment, "If true create this application as a test deployment, which validates that the deployment succeeds and then scales down.")
```

Comment:
Not proposing that, just saying I haven't sorted it out yet and would like to see what the actual "set hooks" command looks like before we do the simpler thing.